docs/sources/explore/index.md

```diff
@@ -155,6 +155,32 @@ Along with metrics, Explore allows you to investigate your logs with the followi
 
 Results of log queries are shown as histograms in the graph and individual logs are displayed below. If the data source does not send histogram data for the requested time range, the logs model computes a time series based on the log row counts bucketed by an automatically calculated time interval and the start of the histogram is then anchored by the first log row's timestamp from the result. The end of the time series is anchored to the time picker's **To** range.
 
+#### Log level
+
+For logs with a **level** label, we use the value of the label to determine log level and update color accordingly. If log doesn't have a level label, we parse the log line to find out if its content matches any of the supported expressions (see below). Log level is always determined by the first match. In a case where Grafana is not able to determine log level, log will be visualised with **unknown** log level.
+
+**Supported log levels and mapping of log level abbreviation and expressions: **
+
+
+|  Supported expressions      | Log level     | Color       |
```

Comment:
Would maybe add the unknown = gray just to also have that color for completeness

Reply:
Yeah, I was thinking about this. But for **Supported expressions**, do you think I should leave it empty?

Reply:
maybe like `* | unknown | grey`? That would make sense to me

Reply:
👍 sounds good. I have added it there. Thanks for the tip!